sources/packages/api/src/route-controllers/institution-locations/institution-locations.controller.ts

```diff
@@ -133,7 +132,7 @@ export class InstitutionLocationsController extends BaseController {
     }
     // get all institution locations.
     const InstitutionLocationData =
-      await this.locationService.getAllInstitutionlocations(
+      await this.locationService.getAllInstitutionLocations(
```

Comment:
Not part of your changes, but here we don`t need the code in lines 125-131 because we can use the institution id from the token directly in line 136. Up to you if you want to handle this with this PR.